scripts/build.sh

```diff
@@ -22,6 +22,7 @@ usage() {
   echo "             -d|--docker-namespace <docker_namespace> - a docker repository beginning with gcr.io"
   echo "             -m|--module           <module_to_build>  - one of {openjdk8, openjdk9}"
   echo "           [ -s|--tag-suffix ]                        - suffix for the tag that is applied to the built image"
+  echo "           [ -p|--project ]                           - GCP test project to use for staging images. If not provided, the default GCP project will be used."
```

Comment:
If the user can provide the project, can we get rid of the `--docker-namespace` because it would just be `gcr.io/${project}`?

Reply:
local build is failing for a while now, it hasn't been introduced here, it was introduced with local-shutdown-test being part of the maven build :( 
See #147 to resolve it strategically, or remove the shutdown-test from the maven build.

Reply:
Ok, I forgot the issue was still not resolve.d

Reply:
The `--docker-namespace` arg refers to the final built image, while `--project` is a project to use for staging only. I think we need both.

Reply:
Do we ever use a different project for staging vs publishing?
Even if that's the case, I think it would make sense to just have `--staging-project` and `--publishing-project`, just for the sake of symmetry.

Reply:
:+1: added --publishing-project and --staging-project.